File: BonaFideClient/src/de/jacobs/university/cnds/bonafide/plus/rest/model/DrawableResult.java
package de.jacobs.university.cnds.bonafide.plus.rest.model;

import com.google.gson.annotations.SerializedName;

/**
 * Moder representing drawable result-objects on map
 * @author Tomas
 *
 */

public class DrawableResult {
	@SerializedName("south_west_latitude")
	private double southWestLatitude=0;
	@SerializedName("south_west_longitude")
	private double southWestLongitude=0;
	
	@SerializedName("north_east_latitude")
	private double northEastLatitude=0;
	@SerializedName("north_east_longitude")
	private double northEastLongitude=0;
	
	/**
	 * Range 0-100
	 */
	@SerializedName("quality")
	private int quality=0; // 0-100
	
	public DrawableResult() {
		
	}

	public double getSouthWestLatitude() {
		return southWestLatitude;
	}

	public void setSouthWestLatitude(double southWestLatitude) {
		this.southWestLatitude = southWestLatitude;
	}

	public double getSouthWestLongitude() {
		return southWestLongitude;
	}

	public void setSouthWestLongitude(double southWestLongitude) {
		this.southWestLongitude = southWestLongitude;
	}

	public double getNorthEastLatitude() {
		return northEastLatitude;
	}

	public void setNorthEastLatitude(double northEastLatitude) {
		this.northEastLatitude = northEastLatitude;
	}

	public double getNorthEastLongitude() {
		return northEastLongitude;
	}

	public void setNorthEastLongitude(double northEastLongitude) {
		this.northEastLongitude = northEastLongitude;
	}

	public int getQuality() {
		if (quality>100) {
			return 100;
		}
		if (quality<0) {
			return 0;
		}
		return quality;
	}

	public void setQuality(int quality) {
		this.quality = quality;
	}
	
	
}

File: BonaFideCentralServer/rest_model/DrawableResult.php
<?php
class DrawableResult {
	public $south_west_latitude;
	public $south_west_longitude;
	public $north_east_latitude;
	public $north_east_longitude;
	public $quality;
	
	function __construct()
    {
        $a = func_get_args();
        $i = func_num_args();
        if (method_exists($this,$f='__construct'.$i)) {
            call_user_func_array(array($this,$f),$a);
        }
    } 
	
	public function __construct5($south_west_latitude,$south_west_longitude,$north_east_latitude,$north_east_longitude,$quality) {
		$this->south_west_latitude=$south_west_latitude;
		$this->south_west_longitude=$south_west_longitude;
		$this->north_east_latitude=$north_east_latitude;
		$this->north_east_longitude=$north_east_longitude;
		$this->quality=$quality;
	}
	
	public function __construct0() {
		
	}
}
?>